lib/decidim/decidim_awesome/awesome.rb

```diff
@@ -227,6 +227,11 @@ module DecidimAwesome
       true
     end
 
+    # Roles for which it is necessary to show admin_accountability
+    config_accessor :user_roles do
```

Comment:
```suggestion
    config_accessor :admin_user_roles do
```
Let's change this name to make it more specific